dace/codegen/targets/rtl.py

```diff
@@ -744,10 +792,12 @@ def unparse_tasklet(self, sdfg: sdfg.SDFG, dfg: state.StateSubgraphView, state_i
     RTL_HEADER = """\
 module {name}
 {parameters}
-( input  ap_aclk   // convention: ap_aclk clocks the design
-, input  ap_areset // convention: ap_areset resets the design
-, input  ap_start  // convention: ap_start indicates a start from host
-, output ap_done   // convention: ap_done tells the host that the kernel has finished
+( input  ap_aclk     // convention: ap_aclk clocks the design, specifically the external ports
+, input  ap_areset   // convention: ap_areset resets the design
+, input  ap_aclk_2   // convention: ap_aclk_2 is the secondary clock, which can be used inside
```

Comment:
From this, it seems that we are fully supporting only two clock cycles.
If this is really the case, I think this is enough for our intended use scenarios, but it should be clear from documentation/config schema

Reply:
Right! Everything else has been set up for generating more than 2, so this has also been changed to generate multiple now.